src/app/pages/MainPage/MainPage.module.css

```diff
@@ -0,0 +1,23 @@
+.body_div {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 10% auto;
+  justify-items: center;
+  grid-template-areas: 'Top' 'Bottom';
+  position: relative;
+}
+.header_div {
+  grid-area: Top;
+  position: fixed;
+}
+.card_area_div {
+  border-top: 1px solid black;
+  grid-area: Bottom;
+  overflow-y: scroll;
+}
+.button_div {
```

Comment:
is it necessary to call it all by its HTML-element? Calling it `header`, `card_area` or `button` / `buttonContainer` would make it easier.